packages/blaze/render_tests.js

```diff
@@ -139,222 +132,6 @@ Tinytest.add("blaze - render - input - checked", function (test) {
-  Tracker.flush();
-  test.equal(imageEl.isMap, false);
-});
-
-// test that we correctly update the HTML5 <ol> reversed boolean property
-Tinytest.add("blaze - render - ordered list - reversed", function (test) {
-  var R = ReactiveVar(null);
-  var div = document.createElement("DIV");
-  materialize(OL({reversed: function () { return R.get(); }}), div);
-  var orderedListEl = div.querySelector('ol');
-  test.equal(orderedListEl.reversed, false);
-  orderedListEl.reversed = true;
-
-  R.set("reversed");
-  Tracker.flush();
-  R.set(null);
-  Tracker.flush();
-  test.equal(orderedListEl.reversed, false);
-});
-
-// test that we correctly update the HTML5 <script> boolean properties
-Tinytest.add("blaze - render - script - defer", function (test) {
-  var R = ReactiveVar(null);
-  var div = document.createElement("DIV");
-  materialize(SCRIPT({src: "", defer: function () { return R.get(); }}), div);
-  var scriptEl = div.querySelector('script');
-  test.equal(scriptEl.defer, false);
-  scriptEl.defer = true;
-
-  R.set("defer");
-  Tracker.flush();
-  R.set(null);
-  Tracker.flush();
-  test.equal(scriptEl.defer, false);
-});
-
-// test that we correctly update the HTML5 <video> boolean properties
-Tinytest.add("blaze - render - video - autoplay", function (test) {
-  var R = ReactiveVar(null);
-  var div = document.createElement("DIV");
-  materialize(VIDEO({autoplay: function () { return R.get(); }}), div);
-  var videoEl = div.querySelector('video');
-  test.equal(videoEl.autoplay, false);
-  videoEl.autoplay = true;
-
-  R.set("autoplay");
-  Tracker.flush();
-  R.set(null);
-  Tracker.flush();
-  test.equal(videoEl.autoplay, false);
-});
-
-Tinytest.add("blaze - render - video - controls", function (test) {
-  var R = ReactiveVar(null);
-  var div = document.createElement("DIV");
-  materialize(VIDEO({controls: function () { return R.get(); }}), div);
-  var videoEl = div.querySelector('video');
-  test.equal(videoEl.controls, false);
-  videoEl.controls = true;
-
-  R.set("controls");
-  Tracker.flush();
-  R.set(null);
-  Tracker.flush();
-  test.equal(videoEl.controls, false);
-});
-
-Tinytest.add("blaze - render - video - loop", function (test) {
-  var R = ReactiveVar(null);
-  var div = document.createElement("DIV");
-  materialize(VIDEO({loop: function () { return R.get(); }}), div);
-  var videoEl = div.querySelector('video');
-  test.equal(videoEl.loop, false);
-  videoEl.loop = true;
-
-  R.set("loop");
-  Tracker.flush();
-  R.set(null);
-  Tracker.flush();
-  test.equal(videoEl.loop, false);
-});
-
-Tinytest.add("blaze - render - video - muted", function (test) {
-  var R = ReactiveVar(null);
-  var div = document.createElement("DIV");
-  materialize(VIDEO({muted: function () { return R.get(); }}), div);
-  var videoEl = div.querySelector('video');
-  test.equal(videoEl.muted, false);
-  videoEl.muted = true;
-
-  R.set("muted");
-  Tracker.flush();
-  R.set(null);
-  Tracker.flush();
-  test.equal(videoEl.muted, false);
-});
-
```

Comment:
Removing `minifier-js` from the `.versions` sounds right to me as `minifier-js` was removed recently in https://github.com/meteor/blaze/pull/236 (specifically, in https://github.com/meteor/blaze/commit/83c4ea7ecd86757c3047b96408a77a695701ca96) and the `.versions` file was not updated accordingly.

However, the https://github.com/meteor/blaze/pull/242/commits/0ded311a13bd0cf0fbf8e19574156cdb7e5ed220 commit currently included (which removes this above test) might not be appropriate to merge.  Reason being that `meteor/meteor` is currently using the [`minifier-babili` branch](https://github.com/meteor/blaze/tree/minifier-babili) of the `blaze` repo to address the problem I mentioned in https://github.com/meteor/meteor/pull/8397#issuecomment-286157853.

Merging that particular revert commit into the `blaze` repo might be counter-productive for Blaze as that test _does_ run successfully under their test-harness, which is running on CircleCI (versus Travis) and with a different test runner than Meteor itself (more capable of the HTML5 test).